app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,24 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return self.hummer_level + 4
+
+    def player_info(self) -> str:
+        return (f""

```

Comment:
There is no need to place a single empty string on this line, let the text begin from the first line instead. Apply this change to all places